src/System.Management.Automation/engine/Modules/SwitchProcessCommand.cs

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Runtime.InteropServices;
+
+using Dbg = System.Management.Automation.Diagnostics;
+
+#if UNIX
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// Implements a cmdlet that allows use of execv API
+    /// </summary>
+    [Experimental(ExperimentalFeature.PSExecFeatureName, ExperimentAction.Show)]
+    [Cmdlet(VerbsCommon.Switch, "Process", HelpUri = "https://replace")]
+    public sealed class SwitchProcessCommand : PSCmdlet
+    {
+        /// <summary>
+        /// Command to execute.
+        /// </summary>
+        [Parameter(Position = 0, Mandatory = false, ValueFromPipeline = true, ValueFromRemainingArguments = true)]
+        public string[] WithCommand { get; set; } = Array.Empty<string>();
```

Comment:
Will we fall again in the old history with argument parsing?

Reply:
I'm not sure what you mean?

Reply:
I mean #1995

Reply:
I don't think we need to do anything regarding that in this cmdlet and should be handled by the parameter binder

Reply:
@mklement0 Do you see any issues with reassembling arguments?